dll/win32/shell32/CDefaultContextMenu.cpp

```diff
@@ -256,9 +256,14 @@ HRESULT CDefaultContextMenu::_DoCallback(UINT uMsg, WPARAM wParam, LPVOID lParam
     return E_FAIL;
 }
 
+
 void CDefaultContextMenu::AddStaticEntry(const HKEY hkeyClass, const WCHAR *szVerb)
 {
     PStaticShellEntry pEntry = m_pStaticEntries, pLastEntry = NULL;
+    WCHAR wszDefault[40];
+    DWORD pdwType, dwSize = _countof(wszDefault);
```

Comment:
Should be `sizeof`, not `_countof` (RegQueryValueEx takes a byte count)
Also, `dwType` would make more sense than `pdwType`.